wpeverest/user-registration
# Refactor - User registration js codes

### All Submissions:






### Changes proposed in this Pull Request:
This PR contains the refactor of User Registration JS codes and files like form-builder js, recaptcha js ,etc.

### How to test the changes in this Pull Request:

1. Checkout this branch.
2. Verify if all the functionality of the plugin is working or not.

### Types of changes:




### Other information:




### Changelog entry

> Refactor - User registration js codes
